src/pages/home/Index.jsx

```diff
@@ -29,50 +33,55 @@ export default function Home() {
     fetchFromParams(BREWDOG_API, queryParamOptions(params)).then(setBeers);
   }, [searchParams]);
 
-  return (
-    <>
-      <br />
-
-      {beers !== null &&
-        beers.map((beer) => (
-          <div key={beer.id}>
-            <h4>{beer.id}</h4>
-            <h3>{beer.name.toUpperCase()}</h3>
-            <p>{beer.tagline.toUpperCase()}</p>
-            <i>
-              <small>{beer.brewers_tips}</small>
-            </i>
+  const width = useWindowResize();
 
-            <br />
-            <br />
-            <br />
+  console.log(width);
 
-            {beer.image_url !== null ? (
-              <img src={beer.image_url} height="235px" width="auto" alt="beer image" />
-            ) : (
-              <img src={genericBottle} height="235px" width="auto" alt="beer image" />
-            )}
-            <br />
-            <Link to={`/${beer.id}`}>See beer details</Link>
-            <br />
-            <hr />
-            <br />
+  return (
+    <>
+      {beers !== null && (
+        <>
+          <div className="cardGroup">
+            {beers.map((beer) => (
+              <Card beer={beer} key={beer.id} />
+            ))}
           </div>
-        ))}
 
-      <ReactPaginate
-        pageCount={Math.ceil(NUM_OF_ITEMS / params.perPage)}
-        pageRangeDisplayed={2}
-        marginPagesDisplayed={2}
-        onPageChange={handlePageChange}
-        containerClassName={'container'}
-        previousLinkClassName={'page'}
-        breakClassName={'page'}
-        nextLinkClassName={'page'}
-        pageClassName={'page'}
-        disabledClassNae={'disabled'}
-        activeClassName={'active'}
-      />
+          {width <= 425 ? (
```

Comment:
I would suggest to not use ternary operator (it is pretty hard to read this way), or ideally move to function which will return this JSX